doc/source/whatsnew/v1.5.0.rst

```diff
@@ -935,7 +935,7 @@ Other
 
 .. ***DO NOT USE THIS SECTION***
 
--
+- Bug in :func:`.assert_index_equal` with ``names=True`` and ``check_order=False`` not checking names not (:issue:`47328`)
```

Comment:
***DO NOT USE THIS SECTION***

yep. this is ok here, in previous release notes the assert_* always go in the other section.

Reply:
checked 1.3 notes to be sure there